packages/boot/test/acceptance/controller.booter.acceptance.ts

```diff
@@ -21,6 +23,15 @@ describe('controller booter acceptance tests', () => {
 
   afterEach(stopApp);
 
+  it('binds package.json', async () => {
```

Comment:
Please move this test to a more appropriate file, maybe create a new test file in `test/acceptance` directory?

It looks wrong to me to see package.json-related tests inside `controller booter acceptance tests`.

(I think this is another clue that package.json should be handled by a new booter.)